packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -4,14 +4,43 @@
 
 import React from 'react';
 import { RouteComponentProps } from '@reach/router';
-import { Localized } from '@fluent/react';
+import { FtlMsg } from 'fxa-react/lib/utils';
 import { ReactComponent as PulseHearts } from './account-verified.svg';
 import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
 
 type ReadyProps = {
   serviceName?: string;
   continueHandler?: Function;
-  viewName: string;
+  viewName: viewNameType;
+};
+
+type viewNameType =
+  | 'signin-confirmed'
+  | 'signin-verified'
+  | 'reset-password-confirmed'
+  | 'reset-password-verified'
+  | 'reset-password-with-recovery-key-verified';
+
+const getTemplateValues = (viewName: viewNameType) => {
```

Comment:
Ignorant TS question... Does this need an explicit return type? Or will TS infer that from the `let templateValues` object below?

Reply:
Not ignorant at all! My initial assumption was that it needed the explicit return type, but I (possibly incorrectly?) believe that it infers from the templateValues object, based on the linter passing it

Reply:
Yeah @millsoper you should be able to remove `{ headerText: string; headerId: string }`, I just double checked and the return type seems to be correct without it 👀 
![image](https://user-images.githubusercontent.com/13018240/209201893-65bbee2d-628c-4c66-9543-7100b2d9a26c.png)